Sources/NIOHTTP2/HTTP2ChannelHandler.swift

```diff
@@ -1078,8 +1082,9 @@ extension NIOHTTP2Handler {
                 return try self.syncMultiplexer()
             }
         } else {
+            let loopBoundSelf = NIOLoopBound(self, eventLoop: self.eventLoop!)
```

Comment:
We can't do this: `NIOLoopBound` preconditions that it's on the right EL on `init` and we know we're not on the EL here as we just checked.

Reply:
Yeah, this is the canonical use of `UnsafeTransfer`.